packages/cli/README.md

```diff
@@ -1,66 +1,478 @@
+- [FAQ](#FAQ)
 
-## Usage
+## Purpose and Vision
 
-The [`yarn`](https://classic.yarnpkg.com/en/docs/install) package is required to use the Redwood CLI.
+Redwood provides a first-class CLI that helps you at every stage of development, from your first commit to your first deploy. And it comes with Redwood, which means no extra software to install!
 
-Be sure to prefix all Redwood CLI commands with `yarn`. For example, `yarn redwood new`.
+Redwood uses [yarn workspaces](https://yarnpkg.com/features/workspaces) to separate your app's sides, generators give you a smooth DX, and Prisma to manage your database. We have a generator for nearly everything, on both sides of the app, from components to functions.
 
-Additionally, you can use `rw` as shorthand for `redwood`. For example, `yarn rw new`.
+Since the CLI is the entry point to Redwood, as Redwood continues to grow, especially as more sides and targets are added, so will the CLI.
 
-## Command line basics
+Check out the [roadmap](#roadmap) to see where we’re headed.
+
+## Package Lead
+
+- [@cannikin](https://github.com/cannikin)
+- [@peterp](https://github.com/peterp)
+
+## Roadmap
+
+### Coming Soon
+
+- [Typescript support](https://github.com/redwoodjs/redwood/issues/523)
+- [Generators refactor (plopjs)](https://github.com/redwoodjs/redwood/issues/653)
+
+### Coming Later
+
+- [Multiple database support](https://github.com/redwoodjs/redwood/issues/507)
+- [Prisma studio](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-studio#overview)
+- [Storybook generators](https://github.com/redwoodjs/redwood/issues/231)
+- [Support for dynamic sides and targets](https://github.com/redwoodjs/redwood/pull/355)
+
+## Contributing
+
+Redwood's CLI is built with [Yargs](http://yargs.js.org/). While you'll want to be familiar with it before contributing, we suggest reading the [Adding a Command](#adding-a-command) section of this guide before consulting the yargs docs since we largely use the advanced api.
+
+### Setting up your local dev environment
+
+<!-- TODO--the other way rob mentioned? -->
+We built a command line tool just for contributing! Check out the top-level [contributing](https://github.com/redwoodjs/redwood/blob/master/packages/cli/README.md) guide to set up your local development environment.
+
+### About the Advanced API
+
+We use the [advanced yargs api](https://github.com/yargs/yargs/blob/master/docs/advanced.md). This means that instead of seeing things written as a method chain, with the `command` method doing most of the work, like:
+
+```javascript
+yargs
+  .command(
+    'get',
+    'make a get HTTP request',
+    function (yargs) {
+      return yargs.option('u', {
+        alias: 'url',
+        describe: 'the URL to make an HTTP request to'
+      })
+    },
+    function (argv) {
+      console.log(argv.url)
+    }
+  )
+  .help()
+  .argv
+```
+
+you'll see the arguments to the `command`  method spread across named, exported constants, like:
```

Comment:
```suggestion
you'll see the arguments to the `command` method spread across named and exported constants, like:
```

Reply:
Thinking we'll just take out "named" here since that part is starting to seem obvious.